tests/acceptance/dcs.cf.sub

```diff
@@ -192,6 +193,9 @@ bundle agent default(filename)
 
       "bundles" slist => bundlesmatching("default:(init|test|check|destroy)");
 
+  files:
+      "$(G.testdir)/."    create => "true";
+
```

Comment:
I had to modify a couple of tests, hopefully everything will be good. However jenkins last run was failing strangely in one of masterfiles tests... Not sure what's the problem, shall trigger again.